Source/MSBuildTools.Unity.NuGet/MSBuildForUnity.csproj

```diff
@@ -27,13 +27,20 @@
     <!-- Major is hard-coded and should be incremented on breaking changes -->
     <MajorVersion>0</MajorVersion>
 
-    <MinorVersion>1</MinorVersion>
+    <!-- Minor version is hard-coded and should be incremented on feature additions that aren't breaking changes.-->
+    <MinorVersion>8</MinorVersion>
 
-    <!-- Revision is defaulted to 0, but if running in the lab where BuildID is specified then default the revision to that value. -->
+    <!-- Revision (semver Patch) version is hard-coded and should be incremented when a bug fix is made.-->
     <RevisionVersion>0</RevisionVersion>
-    <RevisionVersion Condition="'$(BUILD_BUILDID)' != ''">$(BUILD_BUILDID)</RevisionVersion>
-
-    <PackageVersion>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</PackageVersion>
+    
+    <Version>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</Version>
```

Comment:
So now Verison and PakcageVersion will always have 0 for revision? I'm a bit confused on this... won't every CI build have the same overall version now unless you manually change something in this file?